Firestore/core/src/firebase/firestore/timestamp_internal.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_TIMESTAMP_INTERNAL_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_TIMESTAMP_INTERNAL_H_
+
+#include "Firestore/core/include/firebase/firestore/timestamp.h"
+
+namespace firebase {
```

Comment:
I'm not sure about the correct namespace here -- no strong opinion, just pointing it out.

Reply:
I'm not sure either. I put it in firebase since (a) that's where the other timestamp is and (b) under the assumption that the rest of firebase will want to use this for the same purpose (or at least, the rest of firebase that uses timestamp.)